src/GuiBase/BaseApplication.cpp

```diff
@@ -374,7 +374,7 @@ void BaseApplication::framesCountForStatsChanged( uint count ) {
 void BaseApplication::addBasicShaders() {
     using namespace Ra::Engine;
     /// For internal resources management in a filesystem
-    std::string resourcesRootDir = {Core::Resources::getRadiumResourcesDir()};
+    auto resourcesRootDir {*Core::Resources::getRadiumResourcesPath()};
```

Comment:
from https://en.cppreference.com/w/cpp/utility/optional/operator*, `The behavior is undefined if *this does not contain a value.`.
So, the option must be checked first. As the requested shaders are mandatory for the app to function, exit if the path is not found.